images/build/debian-iptables/README.md

```diff
@@ -11,11 +11,11 @@ If you're editing the Dockerfile or some other thing, please bump the `TAG` in t
 ```console
 Build and  push images for all the architectures
 $ make all-push
-# ---> staging-k8s.gcr.io/debian-iptables-amd64:TAG
-# ---> staging-k8s.gcr.io/debian-iptables-arm:TAG
-# ---> staging-k8s.gcr.io/debian-iptables-arm64:TAG
-# ---> staging-k8s.gcr.io/debian-iptables-ppc64le:TAG
-# ---> staging-k8s.gcr.io/debian-iptables-s390x:TAG
+# ---> staging-registry.k8s.io/debian-iptables-amd64:TAG
+# ---> staging-registry.k8s.io/debian-iptables-arm:TAG
+# ---> staging-registry.k8s.io/debian-iptables-arm64:TAG
+# ---> staging-registry.k8s.io/debian-iptables-ppc64le:TAG
+# ---> staging-registry.k8s.io/debian-iptables-s390x:TAG
```

Comment:
I'm not sure these are right

Reply:
I think you're right, but I also don't think it's correct as it currently is. When I run the make file it seems to want to push to `gcr.io/k8s-staging-build-image/debian-iptables-${ARCH}:${TAG}` I'll update the file with the correct reference.